doc/rbd/rbd-mirroring.rst

```diff
@@ -0,0 +1,250 @@
+===============
+ RBD Mirroring
+===============
+
+.. index:: Ceph Block Device; mirroring
+
+RBD images can be asynchronously mirrored between two Ceph clusters.  This
+capability uses the RBD journaling image feature to ensure crash-consistent
+replication between clusters.  Mirroring is configured on a per-pool basis
+within peer clusters and can be configured to automatically mirror all pool
+images within a pool or only a specific subset of images.  Mirroring is
+configured using the ``rbd`` command.  The ``rbd-mirror`` daemon is responsible
+for pulling image updates from the remote, peer cluster and applying them to
+the image within the local cluster.
+
+.. important:: To use RBD mirroring, you must have two Ceph clusters, each
+   running the ``rbd-mirror`` daemon.
+
+Pool Configuration
+==================
+
+The following procedures demonstrate how to perform the basic administrative
+tasks to configure mirroring using the ``rbd`` command.  Mirroring is
+configured on a per-pool basis within the Ceph clusters.
+
+The pool configuration steps should be performed on both peer clusters. These
+procedures assume two clusters, named "local" and "remote", are accessible from
+a single host for clarity.
+
+See the `rbd`_ manpage for additional details of how to connect to different
+Ceph clusters.
+
+.. note:: The cluster name in the following examples corresponds to a Ceph
+   configuration file of the same name (e.g. /etc/ceph/remote.conf).
```

Comment:
Maybe just point to the `running-multiple-clusters` since otherwise I would probably also have to document specifying the client name / id.  I think it starts distracting from core message.